tests/components/vizio/test_init.py

```diff
@@ -37,7 +38,12 @@ async def test_load_and_unload(
     assert await hass.config_entries.async_setup(config_entry.entry_id)
     await hass.async_block_till_done()
     assert len(hass.states.async_entity_ids(MP_DOMAIN)) == 1
+    assert DOMAIN in hass.data
+    assert "apps" in hass.data[DOMAIN]
+    assert isinstance(hass.data[DOMAIN]["apps"], DataUpdateCoordinator)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)